model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -398,6 +385,18 @@ def copy_graph_with_ops(graph: Graph) -> Graph:
 
     # Nodes postprocessing stage in new graph
     for op in new_graph.get_op_nodes():
+        # Call normalize node outputs for restored operations to connect temporary Result operations for disconnected
+        # output ports. We need to do that for correct shape inference. These Result operations will be removed during
+        # IR emitting.For TopK operation we should use specific function TopKNormalizer.normalize_outputs.
+        if op.soft_get('type') != 'TopK':
```

Comment:
`TI` also has own normalization in `postprocessing_op_nodes`

Reply:
Yes, you are right. But it is not output normalization and we need to run it to `TI` too. I will clarify this moment in comments.